snapcraft/meta.py

```diff
@@ -222,10 +218,28 @@ def _wrap_exe(command, basename=None):
     return os.path.relpath(wrappath, snap_dir)
 
 
+def _find_bin(binary, basedir):
+    # If it doesn't exist it might be in the path
+    logger.debug('Checking that {!r} is in the $PATH'.format(binary))
+    script = ('#!/bin/sh\n' +
+              '{}\n'.format(common.assemble_env()) +
+              'which "{}"\n'.format(binary))
+    with tempfile.NamedTemporaryFile('w+') as tempf:
+        tempf.write(script)
+        tempf.flush()
+        try:
+            common.run(['/bin/sh', tempf.name], cwd=basedir)
+        except subprocess.CalledProcessError:
+            raise CommandError(binary)
+
+
 def _wrap_apps(apps):
     for app in apps:
         for k in [k for k in ('command', 'stop-command') if k in apps[app]]:
-            apps[app][k] = _wrap_exe(
-                apps[app][k], '{}-{}'.format(k, app))
-
+            try:
+                apps[app][k] = _wrap_exe(apps[app][k], '{}-{}'.format(k, app))
+            except CommandError as e:
+                raise EnvironmentError(
+                    'The specified command {!r} defined in {!r} does '
+                    'not exist'.format(str(e), app))
```

Comment:
@kyrofa and @elopio 

From the manpage
```
       1      if  one  or  more  specified commands is nonexistent or not exe-
              cutable
```

I'll make the change, too bad since travis already ran :weary: 
To be fair, people using the copy plugin get what they deserve! This would not happen when using a proper build system :wink: